sr_unix/error_return.c

```diff
@@ -105,13 +109,13 @@ void error_return(void)
 		send_msg(VARLSTCNT(3) ERR_JIUNHNDINT, 2, dollar_zstatus.str.len, dollar_zstatus.str.addr);
 	}
 	parent_counted_frame = curframe;	/* Not parent frame for device errors -- still at error counted frame */
-	/* Exit if we are at the bottom of the stack */
+	/* Exit if we are at the bottom of the (entire) stack and this is not CI mode */
 	parent_level = dollar_zlevel() - 1;
-	if ((NULL == parent_counted_frame) || (1 > parent_level))
+	if ((NULL == parent_counted_frame) || ((NULL != parent_counted_frame) && !(SFT_CI & parent_counted_frame->type)
+					       && (1 > parent_level)))
 	{
 		EXIT(dollar_ecode.error_last_ecode);
 	}
-	assert(0 < parent_level);
```

Comment:
This one does not seem to have been responded to.

Reply:
Definitely the former. We allow the parent level to go to 0 now.

Missed this one because comment is on "the other side" and short comments are easily missed in this display.